docs/app-dev/abci-cli.md

```diff
@@ -170,18 +180,18 @@ Try running these commands:
 
 > prepare_proposal "abc"
 -> code: OK
--> log: Succeeded. Tx: abc action: UNMODIFIED
+-> log: Succeeded. Tx: abc
```

Comment:
Sorry, forgot to get back to this one.
This snippet is usually in sync with the contents of "example 1". Did you run the test for example 1 with these changes?

Reply:
Yes. I executed the same commands and the output was slightly different.

Reply:
@williambanfield  had already updated the tests in commit 1a4d9397e9 so I changed the doc just to reflect it.